src/woo_publications/api/tests/utils_tests.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase
+
+from ..utils import underscore_to_camel
+
+
+class UnderscoreToCamelTests(TestCase):
+    def test_single_word_does_not_get_manipulated(self):
+        self.assertEqual(underscore_to_camel("single"), "single")
```

Comment:
please stick to the visual blocks I've pointed out before:

```
setup

act

assert
```

So this would become:

```py
def test_single_word_does_not_get_manipulated(self):
    result = underscore_to_camel("single")

    self.assertEqual(result, "single")
```

this makes the tests so much more readable. When reading/debugging tests, at most I want to use 1% of my brain capacity to decipher the test code itself so that I have 99% to spend on The Actual Problem :tm: